libtransmission/torrent.cc

```diff
@@ -1392,7 +1392,7 @@ void tr_torrentSetDownloadDir(tr_torrent* tor, char const* path)
 
     if (tor->download_dir != path)
     {
-        tor->setDownloadDir(path);
+        tor->setDownloadDir(path, true);
```

Comment:
To avoid further confusions (new torrent / move), maybe add a comment, that the method tr_torrentSetDownloadDir() is called only when adding a new torrent, since it sounds rather generic?

Reply:
It's already documented as such:
https://github.com/transmission/transmission/blob/13ae244eed007cd6e94b15de5f2d97a7131cbb87/libtransmission/transmission.h#L1067-L1070

Reply:
Sorry and thanks!